app/Testing/Eloquent/Users.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Brewmap\Testing\Eloquent;
+
+use Behat\Behat\Context\Context;
+use Behat\Gherkin\Node\TableNode;
+use Brewmap\Eloquent\Profile;
+use Brewmap\Eloquent\User;
+use KrzysztofRewak\Larahat\Helpers\RefreshingDatabase;
+use PHPUnit\Framework\Assert;
+
+class Users implements Context
+{
+    use RefreshingDatabase;
+
+    protected User $user;
+
+    /**
+     * @When there is an user created
+     */
+    public function thereIsAnUserCreated(): void
+    {
+        $this->user = factory(User::class)->create();
+    }
+
+    /**
+     * @When there is an user created:
+     * @When there are users created:
+     */
+    public function thereAreUsersCreated(TableNode $table): void
+    {
+        foreach ($table->getHash() as $user) {
+            factory(User::class)->create($user);
```

Comment:
Could calling factory 'x' times for each user potentially create non-unique faker data (e.g. email) ?

Reply:
This line deals with that, I guess:
https://github.com/brewmap/brewmap/blob/9ae83f9eea54b68d1bf770d71bdc9830d1d1e39b/database/factories/UserFactory.php#L18

Reply:
It sure does when calling it like this:
```
factory(App\User::class, 10)->make();
```
but we are not yet sure in the case above.

If faker->unique makes checks against the database, then it will work, but if it makes checks against a partially generated collection, then it might break eventually.

Reply:
Right, need to investigate this.

Reply:
I added emails to test case, it should fix potential problem.